docs/static/settings-file.asciidoc

```diff
@@ -279,10 +279,23 @@ Values other than `disabled` are currently considered BETA, and may produce unin
 | _N/A_
 
 | `api.auth.basic.password`
-| The password to require for HTTP Basic auth
-Ignored unless `api.auth.type` is set to `basic`.
+a| The password to require for HTTP Basic auth. Ignored unless `api.auth.type` is set to `basic`.
+It should meet default password policy which requires non-empty minimum 8 char string that includes a digit, upper case letter and lower case letter.
+The default password policy can be customized by following options:
+
+* Set `api.auth.basic.password_policy.include.digit` `REQUIRED` (default) to accept only passwords that include at least one digit or `OPTIONAL` to exclude from requirement.
+* Set `api.auth.basic.password_policy.include.upper` `REQUIRED` (default) to accept only passwords that include at least one upper case letter or `OPTIONAL` to exclude from requirement.
+* Set `api.auth.basic.password_policy.include.lower` `REQUIRED` (default) to accept only passwords that include at least one lower case letter or `OPTIONAL` to exclude from requirement.
+* Set `api.auth.basic.password_policy.include.symbol` `REQUIRED` to accept only passwords that include at least one special character or `OPTIONAL` (default) to exclude from requirement.
+* Set `api.auth.basic.password_policy.length.minimum` to a value from 9 to 1024 if you want to require more than the 8 character default setting for passwords.
+
 | _N/A_
 
+| `api.auth.basic.password_policy.mode`
+| Raises either `WARN` or `ERROR` message when password requirements are not met.
```

Comment:
```suggestion
| When the API's Basic auth is enabled and configured with a password whose complexity does not match the policy, either emits a `WARN`-level message to the logs or causes an `ERROR` that prevents Logstash from starting.
```

Reply:
I will follow up this since this does not block FF.